src/store/elementalChat.js

```diff
@@ -104,14 +104,27 @@ export default {
     channels: [],
     currentChannelId: null,
     stats: {},
-    statsLoading: false
+    statsLoading: false,
+    agentHandle: '',
+    needsHandle: false
   },
   actions: {
-    initialize ({ state, dispatch }) {
+    initialize ({ dispatch }) {
       const currentChannelId = window.localStorage.getItem('currentChannelId')
       if (currentChannelId) {
         dispatch('joinChannel', currentChannelId)
       }
+      dispatch('initializeAgent')
+    },
+    initializeAgent ({ dispatch, rootState }) {
+      const tryToGetProfile = () => {
```

Comment:
at some point we might want to create a zome call queue that waits for the conductor to connect, to solve this problem in a more general way.